g4py.sh

```diff
@@ -0,0 +1,51 @@
+package: G4PY
+version: "%(tag_basename)s%(defaults_upper)s"
+source: https://github.com/alisw/geant4
+tag: v4.10.01.p03
+requires:
+  - GEANT4
+  - Python-modules
+  - ROOT
+  - boost
+  - XercesC
+  - opengl
+build_requires:
+  - CMake
+  - "Xcode:(osx.*)"
+env:
+  G4PYINSTALL: "$G4PY_ROOT"
+---
+#!/bin/bash -e
+cp -a $SOURCEDIR/environments/g4py/* $INSTALLROOT
```

Comment:
Hi, you are `cmake`ing and building in the installation directory, this is wrong. Can you do, instead:

```
rsync -a $SOURCEDIR/environments/g4py/ .  # copies everything in the current directory, which is the build one
cmake . -D... -DCMAKE_INSTALL_PREFIX=$INSTALLROOT
```

As said, the "current" directory is the scratch, build directory. If you need to refer to it by variable, its name is `$BUILDDIR`.

The use of `rsync` instead of `cp -a` is consistent with the other recipes.

Note that, contrary to the other comment from @ktf, I am telling you to run `cmake $BUILDDIR`, not `cmake $SOURCEDIR`.